.gitignore

```diff
@@ -5,6 +5,7 @@
 /.pnp
 .pnp.js
 yarn.lock
+package-lock.json
```

Comment:
Hi @batteries76 ,

what's the reasoning behind choosing to ignore `package-lock.json` ?
Did you mean to add this file to `.prettierignore` instead?

Reply:
It's a generated file and should be ignored. It changes when you run `npm install` and so the changes are not useful to see in the diffs. Changes made to dependencies will be reflected in the `package-lock`.

Reply:
I didn't mean ignored generally, it's just that it's a generated file that was showing up in the git diffs. I can see that there are issues with not including it, and ways around them.